db/migrate/migration_utils/legacy_yamler.rb

```diff
@@ -32,33 +32,10 @@
 module Migration
   module LegacyYamler
     ##
-    # Tries to load syck and fails with an error
-    # if it was not installed.
-    # To continue with the affected migrations, install syck with `bundle install --with syck`
+    # Loads the given yaml with syck.
+    # This may be altered to allow an optional require in the future.
     def load_with_syck(yaml)
-      @@syck ||= load_syck
-      @@syck.load(yaml)
-    end
-
-    private
-
-    def load_syck
-      require 'syck'
-      ::Syck
-    rescue LoadError => e
-      abort = -> (str) { abort("\e[31m#{str}\e[0m") }
-      abort.call <<-WARN
-      It appears you have existing serialized YAML in your database.
-
-      This YAML may have been serialized with Syck, which allowed to parse YAML
-      that is now considered invalid given the default Ruby YAML parser (Psych),
-      we need to convert that YAML to be Psych-compatible.
-
-      Use `bundle install --with syck` to install the syck YAML parser
-      and re-run the migrations.
-      WARN
-
-      raise e
+      ::Syck.load(yaml)
```

Comment:
Correct me if I am wrong, but you are still specifying Syck as `require: false`. If I understand this correctly that means you will have to require it manually when you need it, right?